modules/kubernetes-addons/kyverno/kyverno-policies-values.yaml

```diff
@@ -0,0 +1,17 @@
+# -- Pod Security Standard profile (`baseline`, `restricted`, `privileged`, `custom`).
```

Comment:
I think we can remove this default values file - I don't see any improvements over the upstream defaults other than using `restricted` standard over `baseline`

Reply:
`restricted` have some necessary policies which are used by most sites. It is harmless to leave this file for customization.